package-lock.json

```diff
@@ -17,9 +17,11 @@
         "jwt-encode": "^1.0.1",
         "miragejs": "^0.1.42",
         "mockman-js": "^1.0.5",
+        "moment": "^2.29.2",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
         "react-masonry-css": "^1.0.16",
+        "react-moment": "^1.1.2",
```

Comment:
You are using both , `react-moment` and `moment`, any perticular reason?

Reply:
No I forgot to remove react-moment